projects/Unit/APIApproval.Approve.verified.txt

```diff
@@ -325,19 +325,23 @@ namespace RabbitMQ.Client
         event System.EventHandler<RabbitMQ.Client.Events.ConsumerTagChangedAfterRecoveryEventArgs> ConsumerTagChangeAfterRecovery;
         event System.EventHandler<RabbitMQ.Client.Events.QueueNameChangedAfterRecoveryEventArgs> QueueNameChangeAfterRecovery;
         event System.EventHandler<System.EventArgs> RecoverySucceeded;
-        void Abort();
-        void Abort(System.TimeSpan timeout);
-        void Abort(ushort reasonCode, string reasonText);
-        void Abort(ushort reasonCode, string reasonText, System.TimeSpan timeout);
-        void Close();
-        void Close(System.TimeSpan timeout);
-        void Close(ushort reasonCode, string reasonText);
-        void Close(ushort reasonCode, string reasonText, System.TimeSpan timeout);
+        void Close(ushort reasonCode, string reasonText, System.TimeSpan timeout, bool abort);
         RabbitMQ.Client.IModel CreateModel();
         void HandleConnectionBlocked(string reason);
         void HandleConnectionUnblocked();
         void UpdateSecret(string newSecret, string reason);
     }
+    public static class IConnectionExtensions
```

Comment:
Did you forget to add and push this file? It seems so and that explains the compilation error

Reply:
Whoooops 😅 you are most certainly correct. Let me add it...
So the SDK update won't be needed then, but could still be done if wanted.

Reply:
api/ was still ignored from when it was generated. this was why it was missing. Thanks for spotting it.